app/src/main/java/com/android/bookswap/model/map/BookFilter.kt

```diff
@@ -0,0 +1,53 @@
+package com.android.bookswap.model.map
+
```

Comment:
I do have to say, I find it somewhat difficult to ensure my comprehension of the functions below without some aid from comments. It would certainly be well welcome to see some comments added for each function below.

Reply:
Yeah I forgot to add docs but i just commited it right now